pkg/k8shandler/kibana/route.go

```diff
@@ -185,3 +172,50 @@ func (clusterRequest *KibanaRequest) createOrUpdateKibanaConsoleExternalLogLink(
 	}
 	return nil
 }
+
+func (clusterRequest *KibanaRequest) removeSharedConfigMapPre45x() error {
+	cluster := clusterRequest.cluster
+
+	sharedConfig := NewConfigMap("sharing-config", cluster.GetNamespace(), map[string]string{})
+	err := clusterRequest.Delete(sharedConfig)
+	if err != nil && !errors.IsNotFound(err) {
+		return fmt.Errorf("Failure creating Kibana route shared config: %v", err)
```

Comment:
Message is not appropriate for the action being taken.

Reply:
:+1: